name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/MyDividends24Uploader.java

```diff
@@ -47,29 +42,26 @@ public List<Pair<Integer, String>> getPortfolios() throws IOException
         @SuppressWarnings("unchecked")
         JSONArray portfolios = (JSONArray) session.getOrDefault("depots", Collections.emptyList()); //$NON-NLS-1$
 
-        // Maps each portfolio to a pair of its index and its name and returns
-        // the list of pairs.
-        return IntStream.range(0, portfolios.size()).mapToObj(i -> new Pair<>(i, (String) portfolios.get(i))).toList();
+        return portfolios;
     }
 
     @SuppressWarnings("unchecked")
-    public void upload(Client client, CurrencyConverter converter, String portfolioId, Portfolio ppPortfolio)
+    public void upload(Client client, CurrencyConverter converter, String MY24PortfolioID, Portfolio portfolio)
```

Comment:
Please use consistent variables and function names.
MyDividends24Uploader vs. selectedMY24Portfolio vs. MyDividends24 vs. MY24PortfolioID ...

Use "myDividends24XXXXX" as an example.

Reply:
Hi Nirus2000,
in my last Commit I change the Namings all to myDividends24xxxxx as you requested it. 
I want to mention that I will no longer be available for this project so I want to be this approved. I got fired at my Job, what reasons my unavailability.